Smartblock ui: add collapsed "advanced options" section

frecuencialibre opened the pull request:
this PR hides a bunch of settings in the Smartblock edit pane that I've seen confuse users within a collapsible "Advanced options" section:

![screenshot from 2018-12-13 09-29-04](https://user-images.githubusercontent.com/36803137/49948724-9d85da00-feb9-11e8-848b-f8a6e883b655.png)

![screenshot from 2018-12-13 15-07-57](https://user-images.githubusercontent.com/36803137/49967592-f3be4180-fee8-11e8-9241-aadf1c796582.png)

Robbt commented:
This looks good but I noticed there was a lot of spacing between the preview and the advanced tracks dropdown on firefox.
If you remove this line it seems to fix it. I didn't test what impact this had on chrome.
https://github.com/LibreTime/libretime/blob/4752c728b9411b6aa5007b34884cc68e8f9c0c7e/airtime_mvc/public/css/dashboard.css#L593

Here is how it shows up without removing it the min-height of 6em.
![withoutfix](https://user-images.githubusercontent.com/760739/49973206-33455780-ff02-11e8-9ece-7ac50863c6e1.png)
here it is with removing that line
![withfixscreenshot](https://user-images.githubusercontent.com/760739/49973214-3a6c6580-ff02-11e8-8dfb-a7a812e27195.png)

It pushes it down properly when the advanced options is expanded as well. So I'd recommend that change but otherwise I think it is a great improvement in terms of making the interface simpler for people to learn.

frecuencialibre commented:
hopefully the commit i just fixed pushed fixes that spacing issue. that min-height is actually important for smaller screens, so i fixed this in another way.
